src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -104,10 +105,7 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConformBidState
           }
         }
       `
-      const interval = setInterval(() => {
-        metaphysics({ query }).then(this.checkBidPosition.bind(this))
-      }, 1000)
-      this.setState({ intervalToken: interval })
+      metaphysics({ query }).then(this.checkBidPosition.bind(this))
```

Comment:
I changed this because we want to check for the bid result right away. Maybe Gravity has processed everything in the time it takes for us to get to this point? I looked and [the Kiosk code](https://github.com/artsy/eidolon/blob/master/Kiosk/Bid%20Fulfillment/BidCheckingNetworkModel.swift) checks right away.